src/Sentry.Bindings.Native/Sentry.Bindings.Native.csproj

```diff
@@ -0,0 +1,129 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>.NET Bindings for the Sentry Native SDK</Description>
+    <TargetFramework>net7.0</TargetFramework>
+    <SentryNativeSourceDirectory>..\..\modules\sentry-native\</SentryNativeSourceDirectory>
+    <SentryNativeLibraryName>sentry-native</SentryNativeLibraryName>
+    <SentryNativeOutputDirectory>$(MSBuildThisFileDirectory)runtimes\</SentryNativeOutputDirectory>
+    <!-- List of runtime identifiers: https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json -->
+    <NativeLibRelativePath-win-x64>win-x64\native\</NativeLibRelativePath-win-x64>
+    <SentryNativeOutputDirectory-win-x64>$(SentryNativeOutputDirectory)$(NativeLibRelativePath-win-x64)</SentryNativeOutputDirectory-win-x64>
+    <NativeLibRelativePath-linux-x64>linux-x64\native\</NativeLibRelativePath-linux-x64>
+    <SentryNativeOutputDirectory-linux-x64>$(SentryNativeOutputDirectory)$(NativeLibRelativePath-linux-x64)</SentryNativeOutputDirectory-linux-x64>
+    <!-- TODO conditional
+    <SentryNativeEnabled>false</SentryNativeEnabled>
+    <SentryNativeEnabled Condition="'$(TargetPlatformIdentifier)' == 'windows'">true</SentryNativeEnabled> -->
+    <SentryNativeEnabled>true</SentryNativeEnabled>
+  </PropertyGroup>
+
+  <!-- Packaging -->
+  <ItemGroup Condition="$(SentryNativeEnabled)">
+    <!-- Use a separate readme file in the nuget. -->
+    <None Remove="$(MSBuildThisFileDirectory)..\..\README.md" />
+    <None Include="$(MSBuildThisFileDirectory)README.md"
+      Pack="true"
+      PackagePath="" />
+
+    <!-- Don't add the changelog to the nuget. -->
+    <PackageReference Remove="SIL.ReleaseTasks" />
+
+    <!-- Include our transitive build targets in the nuget. -->
+    <None Include="buildTransitive\Sentry.Bindings.Native.targets">
+      <Pack>true</Pack>
+      <PackagePath>buildTransitive\</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <!-- Packaging the native library -->
+  <ItemGroup Condition="'$(CI_PUBLISHING_BUILD)' == 'true' or $([MSBuild]::IsOsPlatform('Windows'))">
+    <NativeLibs Include="$(SentryNativeOutputDirectory-win-x64)$(SentryNativeLibraryName).lib" />
+    <NativeLibs Include="$(SentryNativeOutputDirectory-win-x64)$(SentryNativeLibraryName).pdb" />
+    <None Include="@(NativeLibs)">
+      <Pack>true</Pack>
+      <PackagePath>\runtimes\$(NativeLibRelativePath-win-x64)</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(CI_PUBLISHING_BUILD)' == 'true' or $([MSBuild]::IsOsPlatform('Linux'))">
```

Comment:
It's intentional here though not yet necessary, I've extracted that as a "next step" in the PR description:
> collect both platforms for the nuget package when running in CI

As to why: because we can't build all platforms on a single machine and I don't really want to do the download from a CI job the way I had to do in sentry-unity because that is very painful whenever there's a change, e.g. you want to run different sentry-native version on your branch than what has been compiled as an artifact on the main branch.

The way this project is currently implemented is that it compiles for **your** platform locally and only if necessary (i.e. sentry-native submodule checkout has changed or if there's a change in this `Sentry.Bindings.Native.csproj` file). In CI, it will collect and package for all platforms. At least that's the idea.